controllers/storageclient_controller.go

```diff
@@ -93,9 +95,11 @@ func (r *StorageClientReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&v1alpha1.StorageClient{}).
 		Owns(&v1alpha1.StorageClaim{}).
 		Owns(&batchv1.CronJob{}).
+		Owns(&quotav1.ClusterResourceQuota{}).
```

Comment:
Do we need to wake up on any changes or just for spec changes?
I would suggest adding a Generation Changed predicate here (`predicate.GenerationChangedPredicate{}`)